internal/netconf/interfaces.go

```diff
@@ -82,7 +82,12 @@ func NewIfacesConfigApplier(kind BareMetalType, kb KnowledgeBase, tmpFile string
 // Validate network interfaces configuration. Assumes ifupdown2 is available.
 func (v IfacesValidator) Validate() error {
 	log.Infof("running 'ifup --syntax-check --all --interfaces %s to validate changes.'", v.path)
-	return exec.NewVerboseCmd("ifup", "--syntax-check", "--all", "--interfaces", v.path).Run()
+	err := exec.NewVerboseCmd("ifup", "--syntax-check", "--all", "--interfaces", v.path).Run()
+	if err != nil {
+		log.Errorf("validation of interface configuration failed:%v", err)
+	}
+	// FIXME workaround for centos
+	return nil
```

Comment:
Wouldn't it be better to have an OS check and return `nil` only for Centos error case?